### PR TITLE
cEP-0005: Update to current designs

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

File: cEP-0005.md [hunk 1]
```diff
@@ -113,27 +113,30 @@ class RedundancyBear(LocalBear):
...
-                           "to be detected as a code clone.", int, 20)
+from coalib.bearlib.aspects import Root
+
+Root.new_subaspect(
+   "Redundancy",
+   doc=AspectDocumentation(
+       """
+       This aspect describes redundancy in your source code. Those are
+       usually places where you can strip away source code in order to
+       make your codebase better maintainable but keeping your full
```
Comment (Adrianzatreanu) [inline cEP-0005.md]: better -> more

**Comment by Adrianzatreanu** [comment]
ack 0cf9c8ac4dff997f2d44ef94aba5f74bbaf15263

**Comment by sims1253** [comment]
ack 0cf9c8a

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**